interface/main/tabs/css/userdata.css

```diff
@@ -9,6 +9,6 @@
     font-weight : 600;
 }   
 #userdata .dropdown-menu>li:hover{
-    text-decoration : underline;
-    color: #31b0d5;
+   /* text-decoration : underline;*/
+    background-color: #5cb5f3f2;
 }
```

Comment:
Add a newline at the end of each file that does not have one.  This is a Git file normalization standard, and we try to catch these as they come up.